arangograph/single-sign-on.md

```diff
@@ -9,33 +9,96 @@ title: Single Sign-On (SSO)
+
+6. Replace **ORG_ID** with your organization identifier from the 
+   ArangoGraph Dashboard. To find your organization ID, go to the **User Toolbar**
+   in the top right corner, which is accessible from every view of the Dashboard,
+   and click **My organizations**.
+
+   If, for example, your organization ID is 14587062, here are the values you
+   would use when configuring the SAML settings:
+   - `https://auth.arangodb.com/login/callback?connection=14587062`
+   - `urn:auth0:arangodb:14587062`
+   
+   ![ArangoGraph Organization ID](images/arangograph-organization-id.png)
+7. In the **Attribute Statements** section, add custom attributes as seen in the image below:
+   - email: `user.email`
+   - first_name: `user.firstName`
+   - last_name: `user.lastName`
+   - email_verified: `true`
+   - picture: `user.profileUrl`
+
+   This step consists of a mapping between the ArangoGraph attribute names and
+   Okta attribute names. The values of these attributes are automatically filled
+   in based on the users list that is defined in Okta.
+
+   ![ArangoGraph Okta SAML Attributes](images/arangograph-okta-saml-attributes.png)
+8. Click **Next**.
+9. In the **Configure feedback** section, select **I'm an Okta customer adding an internal app**.
+10. Click **Finish**. The SAML app integration is now created.
+
+### SAML Setup
 
-The SCIM specification expects the following operations so that the SSO system
-can sync the information about user resources in real-time:
+After creating the app integration, you must perform the SAML setup to finalize
+the SSO configuration.
 
-- `GET /Users` - Lists all users.
-- `GET /Users/:user_id` - Get details for a given user ID.
-- `POST /Users` - Invite a new user to ArangoGraph.
-- `PATCH /Users/:user_id` - Update a given user ID.
-- `DELETE /Users/:user_id` - Delete a specified user ID.
+1. Go to the **SAML Signing Certificates** section, displayed under the **Sign On** tab.
+2. Click **View SAML setup instructions**.
 
-Organization administrators can generate an API key for a specific organization.
-The API key consists of a key and a secret. Using this key and secret as the
-Basic Authentication Header in SCIM provisioning, you can access the APIs and
-manage the user resources.
+   ![ArangoGraph Okta SAML Setup](images/arangograph-okta-saml-setup.png)
+3. The setup instructions include the following items:
+   - **Identity Provider Single Sign-On URL**
+   - **Identity Provider Issuer**
+   - **X.509 Certificate**
+4. Copy the IdP settings, download the certificate using the
+   **Download X.509 certificate** button, and share them with the ArangoGraph
+   team via an ArangoGraph Support Ticket in order to complete the SSO
+   configuration.
 
-To learn how to generate a new API key, see the [API Keys](my-account.html#api-keys) section.
+   {% hint 'warning' %}
+   Do not send this information over unencrypted channels, e.g. via normal email.   
+   {% endhint %}
```

Comment:
@Simran-B I know you requested to add it but could I ask that this is removed?

The reasoning is that none of what they'll be sharing with us is "sensitive", the signing certificate is the public certificate. Also, when raising a support ticket they'll be sent over email anyway, so I don't want to give the users the impression that this is "unsafe".

Reply:
Agreed with Simran that we can remove this warning.